source/figures/SConscript

```diff
@@ -0,0 +1,11 @@
+Import('*')
+
+target = ['#source/figures/gdp_educ.lyx']
+source = ['#output/analysis/plots/gdp_educ.eps']
+
+env.Depends(target = target, dependency = source)
```

Comment:
I did not know about this trick for specifying a dependency without specifying a build operation. Neat!